macOS/DuckDuckGo/Tab/TabExtensions/TabCrashRecoveryExtension.swift

```diff
@@ -130,16 +130,16 @@ extension TabCrashRecoveryExtension: NavigationResponder {
             NSUnderlyingErrorKey: NSError(domain: WKErrorDomain, code: terminationReason)
         ])
 
-        Task {
+        attemptTabCrashRecovery(for: error, in: webView)
+
+        Task.detached(priority: .utility) {
```

Comment:
Here (and below) I’m just optimizing pixel firing so that it doesn’t happen on main thread (in the scenario below, especially).